src/main/kotlin/org/opensearch/observability/ObservabilityPlugin.kt

```diff
@@ -81,9 +84,15 @@ class ObservabilityPlugin : Plugin(), ActionPlugin, JobSchedulerExtension {
     ): Collection<Any> {
         PluginSettings.addSettingsUpdateConsumer(clusterService)
         ObservabilityIndex.initialize(client, clusterService)
+        ObservabilityMetricsIndex.initialize(client, clusterService)
         return emptyList()
     }
 
+    override fun onNodeStarted() {
+        ObservabilityIndex.afterStart()
```

Comment:
If we are triggering this method ourselves, why do we want to extend LifecycleListener in ObservabilityIndex class.

Reply:
correct - but it also useful as sort of a marker interface ...